### PR TITLE
Improve styling of panel and buttons

### PR DESCRIPTION
**Pull request by casebenton**
@mythmon r?

I thought it would be good to clean up the styling of the panel a bit. Next week I'll be re-styling using the mockup that Markus will send me, but I figured it would be nice to have it at least look decent for now.

### CODE REVIEW

File: data/recommendation/recWindow.html [hunk 1]
```diff
@@ -4,25 +4,25 @@
...
         any other sites, we recommend you change those as well.
       </p>
-      <div class="buttons">
-        <button id="finished">I've changed my password</button>
-        <button id="no-account">I don't have an account for this site</button>
-        <button id="disable-all">Hide all warnings</button>
+      <div class="button-div">
+        <button id="finished" class="button">I've changed my password</button>
+        <button id="no-account" class="button secondary">I don't have an account</button>
+        <button id="disable-all" class="button secondary">Hide all warnings</button>
```
Comment (mythmon) [inline data/recommendation/recWindow.html]: In all of these you're repeating information. If it matters that the div is a div, just say `div.button`. I'd probably go with a classname of `button-group`, and leave off the `div` part. Same with the buttons. Giving a button a classname of button is redundant. A selector like `.button-group > button` would probably be more appropriate.

Comment (mythmon) [inline data/recommendation/recWindow.html]: After reading more code, I think `.info-box button` would be a better selector.

File: data/recommendation/style.scss [hunk 2]
```diff
@@ -46,32 +48,37 @@ h1 {
...
+  height: 160px;
+  padding: 10px;
   &.hidden {
     display: none;
   }
 }
 
-.complete-box.hidden {
-  display: none;
-}
```
Comment (mythmon) [inline data/recommendation/style.scss]: +1 for unifying `.complete-box` and `.welcome-box` into `.info-box`.

**Comment by casebenton** [comment]
How does it look now? I didn't want to use a decedent `button-group > button` selector because I have one button that is not a child of a button group. I clarified the class name of the button group, because its purpose is to take up the full width of its parent (where conversely, the sync sign up button is not the child of a full width div)

**Comment by mythmon** [comment]
This looks pretty good now.

**Pull request merged**